tests/integration/test_current_branch_validation.py

```
@@ -0,0 +1,284 @@
+                    )
+                else:
+                    # If it's a file, it should be a reusable workflow YAML.
+                    assert target.suffix in (".yml", ".yaml"), (
+                        f"{workflow_file}: local file reference '{uses}' points to '{local_ref}', "
+                        "but it is not a .yml/.yaml file"
+                    )
+        workflow_dir = Path(".github/workflows")
+        
+        for workflow_file in list(workflow_dir.glob("*.yml")) + list(workflow_dir.glob("*.yaml")):
+            with open(workflow_file, 'r') as f:
+                content = f.read()
```

Ruff (0.14.11)

Local variable `content` is assigned to but never used

Remove assignment to unused variable `content`

(F841)